ironfish-cli/src/commands/service/snapshot.ts

```diff
@@ -108,48 +93,23 @@ export default class CreateSnapshot extends IronfishCommand {
     }
     Assert.isNotUndefined(exportDir)
 
-    const blockExportPath = this.sdk.fileSystem.join(exportDir, 'blocks')
-    await this.sdk.fileSystem.mkdir(blockExportPath, { recursive: true })
-
-    this.log('Connecting to node...')
-
-    // TODO: There's a significant slowdown in the export process when running a
-    // full node. This may be due to CPU starvation or message formatting calls
-    // due to the use of node-ipc. We should revisit this in the future to allow
-    // for export without shutting down the node. -- deekerno
     const client = await this.sdk.connectRpc(true)
 
-    const response = client.snapshotChainStream({
-      maxBlocksPerChunk: flags.maxBlocksPerChunk,
-    })
-
-    const { start, stop } = await AsyncUtils.first(response.contentStream())
-    this.log(`Retrieving blocks from ${start} -> ${stop} for snapshot generation`)
+    const chainDatabasePath = this.sdk.fileSystem.resolve(this.sdk.config.chainDatabasePath)
+    const databaseName = this.sdk.config.get('databaseName')
+    const databaseLockPath = this.sdk.fileSystem.join(databaseName, 'LOCK')
 
-    const progress = CliUx.ux.progress({
-      format: 'Retrieving blocks: [{bar}] {value}/{total} {percentage}% | ETA: {eta}s',
-    }) as ProgressBar
-
-    progress.start(stop - start + 1, 0)
-
-    for await (const result of response.contentStream()) {
-      if (result.buffer && result.seq) {
-        const blockFilePath = this.sdk.fileSystem.join(blockExportPath, `${result.seq}`)
-        await fsAsync.writeFile(blockFilePath, Buffer.from(result.buffer))
-        progress.update(result.seq || 0)
-      }
-    }
-
-    progress.stop()
+    const chainInfoResponse = await client.getChainInfo()
+    const blockHeight = Number(chainInfoResponse.content.currentBlockIdentifier.index)
 
     const timestamp = Date.now()
 
     const snapshotFileName = `ironfish_snapshot_${timestamp}.tar.gz`
```

Comment:
Do we need to have multiple snapshots here? What if we have just 1 path and delete the one that exists if it already exists. You'll need to clean up all the snapshots otherwise but having 1 will reduce the need for that.

Reply:
Changed to always use the same default filename with an option to set a different one